frontend/src/pages/SwitchIntergation/SwitchIntergation.tsx

```diff
@@ -0,0 +1,374 @@
+														trackingId="intergation_type-docs-link"
+														//for now redirect to docs. TO DO update it to integration url
+														to={`https://www.highlight.io/docs/general/integrations/${integration.key}-integration`}
+														target="_blank"
+														size="xSmall"
+														kind="secondary"
+														emphasis="low"
+														iconLeft={
+															<IconSolidGlobe />
+														}
+													>
+														{integration.key}
+														.com
+													</LinkButton>
+												</Stack>
+											</Stack>
+											<Box cssClass="justify-self-end">
+												{integration.defaultEnable ? (
+													<Button
+														onClick={() => {
+															setShowConfiguration(
+																false,
+															)
+															setShowDeleteConfirmation(
+																true,
+															)
+														}}
+													>
+														Disconnect
+													</Button>
+												) : (
+													<Button
+														iconRight={
+															<IconSolidArrowSmRight />
+														}
+														onClick={(e) => {
+															e.preventDefault()
+															setShowConfiguration(
+																true,
+															)
+														}}
+													>
+														Connect
+													</Button>
+												)}
+											</Box>
+										</Box>
+									</BorderBox>
+								)
+							})}
+							<Box mt="24">
+								<Tabs<IntegrationSettingsTabs>
+									selectedId={
+										(params?.get('tab') ||
+											IntegrationSettingsTabs.Description) as IntegrationSettingsTabs
+									}
+									onChange={(id) => {
+										navigate(
+											`${location.pathname}?tab=${id}`,
+										)
+									}}
+								>
+									<Tabs.List>
+										<Tabs.Tab
+											id={
+												IntegrationSettingsTabs.Description
+											}
+										>
+											Description
+										</Tabs.Tab>
+										<Tabs.Tab
+											id={
+												IntegrationSettingsTabs.Configure
+											}
+										>
+											Configure
+										</Tabs.Tab>
+									</Tabs.List>
+									<Box mt="24">
+										<Tabs.Panel
+											id={
+												IntegrationSettingsTabs.Description
+											}
+										>
+											<Stack>
+												{/* TO DO  Need update this with real description*/}
+												<Box
+													borderRadius="8"
+													border="divider"
+													style={{ minHeight: 300 }}
+													background="secondarySelected"
+												></Box>
+												<Text color="secondaryContentText">
+													Bring your highlight.io
+													comments and alerts to{' '}
+													{integration.name} as
+													messages.
+												</Text>
+											</Stack>
+										</Tabs.Panel>
+										<Tabs.Panel
+											id={
+												IntegrationSettingsTabs.Configure
+											}
+										>
+											{/*TO DO: This i am not sure. have to check with them*/}
```

Comment:
https://www.loom.com/share/072d634584ce4f8fa5861a2baeb43733

sharing with @julian-highlight  to include designs for other types of configurations
![Screenshot from 2024-12-09 16-48-50](https://github.com/user-attachments/assets/7b8aade2-b5bc-401d-88a8-a0801de63786)
![Screenshot from 2024-12-09 16-48-41](https://github.com/user-attachments/assets/99a08a7d-1262-44d3-a19b-22390f5a95ee)